lib/charms/layer/nodejs.py

```diff
@@ -12,11 +12,10 @@ def node_dist_dir():
     Returns:
     Absolute string of node application directory
     """
-    storage_id = hookenv.storage_list('app')[0]
-    return hookenv.storage_get('location', storage_id)
```

Comment:
Thanks, if there is a way to migrate upgrades to using the new location I would be ok with that. I'm also going to rebuild the ghost charm locally to see if something in reactive changed and now causing the storage to fail